### PR TITLE
Feature/teachers pets

### PR DESCRIPTION
**Pull request by KyleWong2510**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type
 
 
Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe): 

## What is the current behavior?

- No display for high scores

## What is the new behavior?

- Displays high scores on the click of a button
- Runs a GET method to populate the scores
- Fn iterates through the scores to display any in the list

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: src/GameOver/GameOver.js [hunk 1]
```diff
@@ -13,6 +14,7 @@ const GameOver = (props) => {
         <p>Score: {props.score}</p>
         {props.lives > 0 && <p>Lives Left: {props.lives}</p>}
         <Link to='/'><button className='nes-btn' type='submit' onClick={props.reset}>Home</button></Link>
+        <Link to='/nerdz'><button className='nes-btn' onClick={props.getTeachersPets}>Teachers Pets</button></Link>
```
Comment (lilianaweimer) [inline src/GameOver/GameOver.js:17]: Nice! /nerdz :)

File: src/TeachersPets/TeachersPets.js [hunk 2]
```diff
@@ -1,18 +1,37 @@
...
-                    </ul>
-            <button>Home</button>
-        </section>
+      <section className='scoreboard'>
+        <h2 className='scoreboard-title'>Teacher's Pets:</h2>
+        <ul className='scoreboard-list'>
+          {pets}
+        </ul>
+        <Link to='/'><button className='nes-btn'>Home</button></Link>
+      </section>
```
Comment (lilianaweimer) [inline src/TeachersPets/TeachersPets.js:21]: This looks great!

File: src/TeachersPets/TeachersPets.css [hunk 3]
```diff
@@ -1,12 +1,13 @@
 .scoreboard {
```
Comment (Billwilke42) [inline src/TeachersPets/TeachersPets.css:1]: nice work on the scoreboard!

File: src/GameOver/GameOver.js (hunk 1 above)
Comment (Billwilke42) [inline src/GameOver/GameOver.js:17]: Love the /nerdz

**Pull request merged**